gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst

```diff
@@ -0,0 +1,124 @@
+.. _rfc-79:
+
+=============================================================
+RFC 79: Listing of Service Providers on GDAL website
+=============================================================
+
+============== ============================
+Author:        Daniel Morissette, Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2021-Feb-22
+Last updated:  2021-Feb-22
+Status:        Development
+============== ============================
+
+1. Motivation
+=============
+
+GDAL is developed and supported by a rich ecosystem of businesses and
+individuals around the world but we do not do a good job of promoting those
+service providers and making them easy to find by our users at the moment.
+
+In this RFC, we propose a plan to address this by adding a Service Providers
+page to the GDAL website.
+
+2. Proposed plan
+================
+
+A new Service Providers page will be added to the Sphinx documentation.
+
+The service providers will be grouped in three categories. The descriptions
+below should make it relatively easy to determine in which group a given
+organization belongs.
+
+Each qualified organization who wishes to be listed is responsible for
+adding themselves to the right category in the page through a pull
+request (or a direct commit in case of core committers). We are intentionally
+not going to provide instructions here for preparing the pull request as
+it is expected that a qualified service provider should have people on
+staff who can figure it out themselves. Yes, we intentionally
+try to keep the bar at a minimum level.
+
+A PSC vote is NOT required in order to add a new service provider,
+i.e. any core committer can validate and commit the pull
+request (or their own entry) but in case of doubts or disagreements on the
+interpretation of the rules, the GDAL PSC will be the ultimate authority
+to resolve the question.
+
+Organizations in each category are displayed in randomized order (except
+in the PDF docs which are static). However entries should be added to the
+source file in alphabetical order (for PDF docs purpose).
+The output will be similar to
+https://mapserver.org/community/service_providers.html
+
+2.1 Core Contributors
+---------------------
+
+Core Contributors are essentially the organizations that have one or more
+Core Committer (a person with merge rights in the GDAL GitHub repository)
+and/or PSC members as part of their staff and who provide
+GDAL related services to customers.
+
+To qualify, the Core Committers and/or PSC members must be directly employed
+by the organization and not simply part time contractors.
+
+They get to add a 200px wide logo with a descriptive text outlining their qualifications and services.
```

Comment:
Should there be a cap for the max height here and below?

Reply:
I've changed to "200 px max high or wide logo" and similarly for the other categories